lightning/src/ln/msgs.rs

```diff
@@ -89,6 +89,18 @@ pub struct ErrorMessage {
 	pub data: String,
 }
 
+/// An warning message to be sent or received from a peer
+#[derive(Clone, Debug, PartialEq)]
+pub struct WarningMessage {
```

Comment:
Yea, I'm not sure if it was specifically to avoid conflicts or just because `Error` seems like something that would be used as an `Error` vs `ErrorMessage` is pretty clear. The same logic applies for `Warning`, IMO, but I also don't have particularly strong feelings.